disable_warnings.sh

```diff
@@ -0,0 +1,38 @@
+#floriantschopp/schneith, 07/2019
+#!/bin/bash
+
+# Check for arguments
+if [ -z "$1" ]; then
+  echo "Disable the warnings generated by the eigen core headers by adding pragma's to all header files.";
+  echo -e "Usage: disable_warnings.sh CATKIN_DEVEL_SPACE/EIGEN_SYSTEM_INCLUDE"
+  exit -1
+fi
+
+DEVEL_SPACE=$1
+EIGEN_INCLUDES_DIR=$1/include/eigen3
+CURR_DIR=$(pwd -P)
+TEMPLATE_DIR=$(pwd -P)/template
+cd $EIGEN_INCLUDES_DIR
+CORE_DIR=$(pwd -P)/Eigen/src/Core
+cd $CURR_DIR
+
+echo "Attempt to change headers in $CORE_DIR"
+
+# Now check if gtsam headers are installed and the path is correct
+if [ ! -d "$EIGEN_INCLUDES_DIR" ] || [ ! -f "$CORE_DIR/ArrayBase.h" ]; then
+  echo "Could not find eigen headers in the specified devel space!";
+  exit -1
+fi
+
+# Check wheter the warnings are already disabled
+ALREADY_IGNORED=$(cat $CORE_DIR/ArrayBase.h | grep "pragma GCC diagnostic ignored \"-Wint-in-bool-context\"" | wc -l)
```

Comment:
Hint: The `cat` is superfluous here since you can simply add the file argument to the `grep`.